packages/utilities/psammead-locales/moment/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  rules: {
+    // IE11 gives syntax error for trailing commas on functions and Moment
+    // locales are not transpiled
+    'comma-dangle': ['error', {
```

Comment:
Can we use the `es5` setting? https://prettier.io/docs/en/options.html#trailing-commas

Reply:
That's a much better idea! Updated.